docs/src/man/subsets.md

```diff
@@ -6,7 +6,7 @@ The `DataArray` type is meant to behave like a standard Julia `Array` and tries
 
 One dimensional `DataArray`:
 
-```julia
```

Comment:
The following gist passes doctesting:

https://gist.github.com/MichaelHatherly/b06b3a40bcecf4abb3e0

Note the need to fully qualify `DataFrames.DataFrame` and other similar names. You can use a `{meta}` block with a `DocTestSetup = ` value to run some code before each doctest.

Also, if doctesting is *really* having trouble you can disable is with

```julia
makedocs(
    doctest = false
)
```

which will just print a warning that it's been disabled.